CRITICAL - 2021-12-26 00:33:31 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('7', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(210): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 00:34:04 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('7', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(210): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 00:34:19 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('7', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(210): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 00:35:15 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('7', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(210): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 00:41:09 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('7', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(212): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 00:48:58 --> Undefined offset: 1
#0 D:\project\login_v414\app\Controllers\Admin\Manuser.php(206): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined offse...', 'D:\\project\\logi...', 206, Array)
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#6 {main}
CRITICAL - 2021-12-26 03:02:27 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('7', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(215): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 03:06:05 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('7', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(215): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 03:07:03 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('14', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(215): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 05:28:01 --> Undefined variable: color
#0 D:\project\login_v414\app\Views\tabel\menu.php(20): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'D:\\project\\logi...', 20, Array)
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\View\View.php(214): include('D:\\project\\logi...')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\View\View.php(432): CodeIgniter\View\View->render('tabel/menu', NULL, true)
#4 D:\project\login_v414\app\Views\admin\kelolamenu.php(10): CodeIgniter\View\View->include('tabel/menu')
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\View\View.php(214): include('D:\\project\\logi...')
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\Common.php(1092): CodeIgniter\View\View->render('admin/kelolamen...', Array, true)
#8 D:\project\login_v414\app\Controllers\Admin\Manmenu.php(23): view('admin/kelolamen...', Array)
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manmenu->index()
#10 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manmenu))
#11 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#13 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#14 {main}
CRITICAL - 2021-12-26 08:00:07 --> Undefined variable: data
#0 D:\project\login_v414\app\Views\user\reset_passwrd.php(15): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'D:\\project\\logi...', 15, Array)
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\View\View.php(214): include('D:\\project\\logi...')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\View\View.php(217): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\Common.php(1092): CodeIgniter\View\View->render('user/reset_pass...', Array, true)
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(155): view('user/reset_pass...', Array)
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->reset_password()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 09:55:53 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('14', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(232): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 09:58:20 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('1', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(232): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 09:58:31 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('1', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(232): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:00:43 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('1', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(232): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:05:43 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('1', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(232): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:11:14 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('4', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(228): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:12:32 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('1', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(228): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:16:25 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('4', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(228): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:16:47 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('4', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(228): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:17:47 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('14', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(228): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:19:28 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('14', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(228): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:22:18 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('14', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(228): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:22:53 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('6', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(228): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:24:17 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('14', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(228): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:29:35 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('14', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(233): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:30:04 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('14', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(233): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
CRITICAL - 2021-12-26 10:31:54 --> There is no data to update.
#0 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(1589): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
#1 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(832): CodeIgniter\BaseModel->transformDataToArray(Array, 'update')
#2 D:\project\login_v414\vendor\codeigniter4\framework\system\Model.php(646): CodeIgniter\BaseModel->update(Array, Object(App\Entities\Output))
#3 D:\project\login_v414\vendor\codeigniter4\framework\system\BaseModel.php(656): CodeIgniter\Model->update('14', Object(App\Entities\Output))
#4 D:\project\login_v414\app\Controllers\Admin\Manuser.php(233): CodeIgniter\BaseModel->save(Object(App\Entities\Output))
#5 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(802): App\Controllers\Admin\Manuser->ubah_level()
#6 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Manuser))
#7 D:\project\login_v414\vendor\codeigniter4\framework\system\CodeIgniter.php(317): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\project\login_v414\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 D:\project\login_v414\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('D:\\project\\logi...')
#10 {main}
